packages/account-v2/src/containers/LoginHistoryTable/LoginHistoryTableCard.tsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { LoginHistory } from '@deriv/api-types';
+import { Text } from '@deriv-com/ui';
+import { formattedLoginHistoryData } from '../../utils';
+
+type TLoginHistoryProps = { loginHistory: LoginHistory };
+
+export const LoginHistoryTableCard = ({ loginHistory }: TLoginHistoryProps) => {
+    const formattedLoginHistory = formattedLoginHistoryData(loginHistory);
+
+    return (
+        <div className='inline-flex flex-col items-start gap-16 w-full'>
+            {formattedLoginHistory.map(item => (
+                <div
+                    className='shadow-md p-16 rounded-lg bg-[var(--system-light-8-primary-background,#FFF)] w-full'
+                    key={item.ip}
+                >
+                    <div className='flex gap-40'>
+                        <div className='justify-left flex flex-col items-start gap-8 row-span-3'>
+                            {[
+                                { label: 'Date and time', value: item.date },
+                                { label: 'Browser', value: item.browser },
+                                { label: 'IP Address', value: item.ip },
```

Comment:
Move this out and make it a const

Reply:
We already have this in LoginHistoryTable, cant reuse ?

const header = [
    {
        header: 'Date and Time',
    },
    {
        header: 'Action',
    },
    {
        header: 'Browser',
    },
    {
        header: 'IP Address',
    },
    {
        header: 'Status',
    },
];

Reply:
there is a value ( which we are getting from formatted data component ) also along with the labels that we need to display, how exactly do we reuse the headers here in this case ?